src/client/src/components/CollapsibleInfoBox/index.tsx

```diff
@@ -0,0 +1,65 @@
+import classnames from "classnames";
+import * as React from "react";
+import { ReactComponent as Down } from "../../assets/i-collapsibleDown.svg";
+import { ReactComponent as Up } from "../../assets/i-collapsibleUp.svg";
+
+import { injectIntl, InjectedIntlProps } from "react-intl";
+import styles from "./styles.module.css";
+import { KEY_EVENTS } from "../../utils/constants";
+
+interface IProps {
+  question: string;
+  answer: string;
+  initialAnswerShownState?: boolean;
+}
+
+type Props = IProps & InjectedIntlProps;
```

Comment:
Which prop is of type `InjectedIntlProps`? If there is none, we can probably remove this.

Reply:
Great catch, removed :)